CI/org.mudlet.mudlet.yml

```diff
@@ -0,0 +1,190 @@
+      - >-
+        ./configure --with-lua=/app --prefix=/app
+        --with-lua-include=/app/include --force-config
+      - make build
+      - make install
+    build-options:
+      strip: false
+      no-debuginfo: true
+      no-debuginfo-compression: true
+    sources:
+      - type: archive
+        url: 'https://github.com/luarocks/luarocks/archive/refs/tags/v3.9.2.tar.gz'
+        sha256: a0b36cd68586cd79966d0106bb2e5a4f5523327867995fd66bee4237062b3e3b
+    cleanup:
+      - /bin/luarocks*
+      - /share/luarocks
+      - /etc/luarocks*
+
+  - name: sqlite3
+    config-opts:
+      - CFLAGS=-O2 -DSQLITE_ENABLE_FTS4 -DSQLITE_ENABLE_RTREE
+    sources:
+      - type: archive
+        url: 'https://www.sqlite.org/2021/sqlite-autoconf-3360000.tar.gz'
+        sha256: bd90c3eb96bee996206b83be7065c9ce19aef38c3f4fb53073ada0d0b69bbce3
+    cleanup:
+      - /bin/sqlite3
+
+  - name: pcre8
+    config-opts:
+      --enable-utf --enable-unicode-properties --prefix=/app
+    sources:
+      - type: archive
+        url: 'https://sourceforge.net/projects/pcre/files/pcre/8.45/pcre-8.45.tar.bz2'
+        sha256: 4dae6fdcd2bb0bb6c37b5f97c33c2be954da743985369cddac3546e3218bffb8
+    cleanup:
+    - /bin/pcre-config
+    - /bin/pcregrep
+    - /bin/pcretest
+
+  - name: boost
+    buildsystem: 'simple'
+    build-commands:
+      # we don't need to build boost, only using it for headers
+      - cp -r * /app/include
+    sources:
+      - type: archive
+        url: 'https://sourceforge.net/projects/boost/files/boost/1.77.0/boost_1_77_0.tar.bz2'
+        sha256: fc9f85fc030e233142908241af7a846e60630aa7388de9a5fafb1f3a26840854
+
+  - name: luarucks-mods
+    buildsystem: simple
+    build-options:
+      build-args:
+        - '--share=network'
+    build-commands:
+      - luarocks install --tree="/app" lua-yajl YAJL_INCDIR="/app/include" YAJL_LIBDIR="/app/lib" LUA_INCDIR="/app/include"
+      - luarocks install --tree="/app" luautf8
+      - luarocks install --tree="/app" LuaFileSystem
+      - luarocks install --tree="/app" lua-zip ZIP_INCDIR="/app/include" ZIP_LIBDIR="/app/lib" LUA_INCDIR="/app/include"
+      - luarocks install --tree="/app" LuaSQL-SQLite3 SQLITE_INCDIR="/app/include"
+      - luarocks install --tree="/app" lrexlib-pcre PCRE_INCDIR="/app/include"
+
+  - name: glu
+    config-opts:
+      - --disable-static
+    sources:
+      - type: archive
+        url: https://ftp.osuosl.org/pub/blfs/conglomeration/glu/glu-9.0.2.tar.xz
+        sha256: 6e7280ff585c6a1d9dfcdf2fca489251634b3377bfc33c29e4002466a38d02d4
+    cleanup:
+      - /include
+      - /lib/*.a
+      - /lib/*.la
+      - /lib/pkgconfig
+
+  - name: mudlet
+    buildsystem: cmake-ninja
+    config-opts:
+      # pending https://github.com/Mudlet/Mudlet/pull/6493
+      # - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DCMAKE_BUILD_TYPE=Release
+    sources:
+      - type: git
+        url: https://github.com/Mudlet/Mudlet.git
+        branch: development
```

Comment:
I'm not yet clear on what the flatpak best practices here are.

Reply:
Maybe we'd want a "Release" pack following official release versions and then a second that follows development each night that it changes - i.e. the same as our PTBs...

Reply:
I'm not sure what do you mean, rephase?

It doesn't seem practical to keep modifying the file after every release... although in hindsight we can, because we already have https://github.com/Mudlet/Mudlet/blob/development/.github/repo-metadata.yml which tracks this.

But it would not be practical for PTBs.